### ISSUE TITLE
Wildcards in the Navigator

### ISSUE THREAD

**Issue by ghost** [MisterJulsen/Create-Train-Navigator#10]
**Is your feature request related to a problem? Please describe.**
A station with many station blocks which always begin with the same name is annoying to add to a tag, so I would to have wildcards in the Navigator.

**Describe the solution you'd like**
When typing in a station name, you can use "*" to get all stations with the same beginning/end after/before the "*"

**Describe alternatives you've considered**
-

**Additional context**
-

![image](https://github.com/MisterJulsen/Create-Train-Navigator/assets/59450459/0480de44-f94d-4707-96fb-32deff137926)

**Comment by stacode123** [comment]
Yes please as I have separate names for every track in my stations

**Comment by DanCoddee** [comment]
I may be misunderstanding your problem @matteodev8, but  click the settings button -> Train station tags. Then create a station called Central station, and add all the track stations under different platforms.
When you search for Central station it will include all the track stations for central station in the results.
